src/community/__screenshot_tests__/advanced-data-card-screenshot-test.tsx

```diff
@@ -41,6 +41,22 @@ test.each`
     expect(image).toMatchImageSnapshot();
 });
 
+test.each`
+    extra
+    ${3}
+`('Advanced Data Card extras: $extra no divider', async ({extra}) => {
+    await openStoryPage({
+        id: 'community-advanceddatacard--default',
+        device: 'MOBILE_IOS',
+        args: {extra, noExtraDivider: true},
```

Comment:
No need for the `test.each`. Just set:

`args: {extra: 3, noExtraDivider: true},`